chia/cmds/cmds_util.py

```diff
@@ -59,6 +62,7 @@ async def validate_client_connection(
     root_path: Path,
     fingerprint: Optional[int],
     login_to_wallet: bool,
+    catch_errors: bool = True,
```

Comment:
this parameter name is not intuitive to me. `True` means that we'll *ignore* errors and just log them (and not shut down gracefully). Note that we *catch* the error either way.

It seems the default should be to shut down gracefully (i.e. call `await rpc_client.await_closed()`).

I'm guessing there's some subtle context I'm unfamiliar with, making ungraceful shutdown making sense.

Reply:
good suggestion, ill do that.